src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs

```diff
@@ -51,6 +51,14 @@ public async Task StoreConsent(ConsentArtefactRepresentation consentArtefact)
                     notification.Status,
                     notification.ConsentId);
                 await consentRepository.AddAsync(consent);
+                var cmSuffix = consent.ConsentArtefact.ConsentManager.Id;
+                var gatewayResponse = new GatewayRevokedConsentRepresentation(
```

Comment:
Why `GatewayRevokedConsentRepresentation`? This is not the case of revoke.

Reply:
renamed